content/guides/models/registry/configure_registry.md

```diff
@@ -7,24 +7,52 @@ title: Configure registry access
+Add, remove, and manage registry roles for teams and individual users in a registry by configuring a registry's settings.
 
 {{% alert %}}
-Only registry admins can [restrict visibility]({{< relref "#restrict-visibility-to-a-registry" >}}), [configure user roles]({{< relref "#configure-user-roles-in-a-registry" >}}), or [remove users]({{< relref "#remove-a-user-from-a-registry" >}}) from registries in an organization.
+Only registry administrators can [restrict visibility]({{< relref "configure_registry.md#restrict-visibility-to-a-registry" >}}), [configure registry roles]({{< relref "configure_registry.md#configure-registry-roles" >}}), [add users]({{< relref "configure_registry.md#add-a-user-from-a-registry" >}}), or [remove users]({{< relref "configure_registry.md#remove-a-user-from-a-registry" >}}) from a registry.
 {{% /alert %}}
 
-## Registry roles permissions
 
-A user within an organization can have different roles, and therefore permissions, for each registry in their organization.
+## Add a user or a team to a registry
+
+Registry administrators can add individual users or entire teams to a registry. To add a user or team to a registry:
+
+1. Navigate to the Registry App at https://wandb.ai/registry/.
+2. Select the registry you want to add a user or team to.
+3. Click on the gear icon on the upper right hand corner to access the registry settings.
+4. Navigate to the **Registry access** section.
+5. Click on the **Add access** button.
+6. Add a user name, email, or the name of a team to the **Include users and teams** field.
+7. Click **Add access**.
+
+W&B automatically assigns users a `Viewer` role by default. To edit a user's role, see [Configure user roles in a registry]({{< relref "configure_registry.md#configure-user-roles-in-a-registry" >}}).
+
+## Remove a user or team from a registry
+Registry administrators can remove individual users or entire teams to a registry. To remove a user or team from a registry:
+
+1. Navigate to the Registry App at https://wandb.ai/registry/.
+2. Select the registry you want to remove a user from.
+3. Click on the gear icon on the upper right hand corner to access the registry settings.
+4. Navigate to the **Registry access** section and type in the username, email, or team you want to remove.
+5. Click the **Delete** button.
+
+
+
+## Registry role permissions
+
+Each user in a registry has a specific *registry role*. Registry roles determine what users can do in a given registry.
+
+Registry administrators can assign or modify team and user roles in a registry. See [Configure user roles in a registry]({{< relref "configure_registry.md#configure-user-roles-in-a-registry" >}}) for more information.
```

Comment:
```suggestion
A registry admin can assign or modify roles for users and teams in a registry. See [Configure user roles in a registry]({{< relref "#configure-user-roles-in-a-registry" >}}) for more information.
```

Reply:
Are we using "admin" or "administrators"?

Reply:
I had this question too

Reply:
The winner is "admin"!